libs/eavmlib/src/port.erl

```diff
@@ -19,11 +19,38 @@
 % SPDX-License-Identifier: Apache-2.0 OR LGPL-2.1-or-later
 %
 
+%%-----------------------------------------------------------------------------
```

Comment:
I'd actually be okay if everything in port was `@hidden`, since this is really an internal API.  Curious what others think.

Reply:
I realize is really internal, but because it is so fundamental to the operation of port drivers, I thought it should be documented to make it easier for downstream driver developers.